meta-dts-distro/recipes-bsp/txe-secure-boot/smmstoretool_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "Offline SMMSTORE variable modification too"
+LICENSE = "GPL-2.0-or-later"
+
+SRC_URI = " \
+git://github.com/coreboot/coreboot.git;branch=main;protocol=https \
+file://a5df001.diff;patchdir=${WORKDIR}/git \
```

Comment:
I prefer adding at least 4 spaces here when we break lines. This is rather common if you look at other recipes, to have some indentation here.

Reply:
My bad; fixed 308b71460416f4da9cfb6027e82cf9654487a592